husky_components/search/main.js

```diff
@@ -22,8 +21,8 @@ define([], function() {
 
     var templates = {
             skeleton: [
-                '<a class="fa-search fa-flip-horizontal search-icon" href="#"></a>',
-                '<a class="fa-times-circle remove-icon" href="#"></a>',
+                '<span class="fa-search fa-flip-horizontal search-icon"></span>',
+                '<span class="fa-times-circle remove-icon"></span>',
```

Comment:
thats also weird ... but i will test it! like i said for me its not an anchor like it is used here because we don't change the URL or do any routing. a button would be the semantic right at this point